config/kibana.yml

```diff
@@ -181,3 +181,6 @@
 # Maximum number of documents loaded by each shard to generate autocomplete suggestions.
 # This value must be a whole number greater than zero. Defaults to 100_000
 #unifiedSearch.autocomplete.valueSuggestions.terminateAfter: 100000
+
+feature_flags.overrides:
+  lens.enable_esql: true
```

Comment:
Please, remember to remove this before merging

Reply:
this was just added in the commit named `revert before merging` to check once again that CI is passing when this flag is on

Reply:
Thanks for confirming! I'm sure that Core won't be needed when this is removed :)